test/unit/builtin/functions/array/array_valuesTest.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * PHPRuntime - PHP environment runtime components
+ * Copyright (c) Dan Phillimore (asmblah)
+ * https://github.com/uniter/phpruntime/
+ *
+ * Released under the MIT license
+ * https://github.com/uniter/phpruntime/raw/master/MIT-LICENSE.txt
+ */
+
+'use strict';
+
+var expect = require('chai').expect,
+    arrayExtension = require('../../../../../src/builtin/functions/array'),
+    sinon = require('sinon'),
+    CallStack = require('phpcore/src/CallStack'),
+    NullValue = require('phpcore/src/Value/Null').sync(),
+    PHPError = require('phpcommon').PHPError,
+    ValueFactory = require('phpcore/src/ValueFactory').sync(),
+    ArrayValue = require('phpcore/src/Value/Array').sync(),
+    IntegerValue = require('phpcore/src/Value/Integer').sync(),
+    KeyValuePair = require('phpcore/src/KeyValuePair');
+
+describe('PHP "array_values" basic-level builtin function', function () {
```

Comment:
I don't think we need to mention basic-level here, as this will just be the "full" implementation 😄